safe_transaction_service/history/services/index_service.py

```diff
@@ -388,6 +399,38 @@ def fix_out_of_order(
         SafeLastStatus.objects.filter(address=address).delete()
         logger.info("[%s] Ended fixing out of order", address)
 
+    def process_address(self, safe_address: ChecksumAddress) -> int:
+        """
+        Process all the pending `InternalTxDecoded` for a Safe
+
+        :param safe_address:
+        :return: Number of `InternalTxDecoded` processed
+        """
+
+        # Check if a new decoded tx appeared before other already processed (due to a reindex)
+        if InternalTxDecoded.objects.out_of_order_for_safe(safe_address):
+            logger.error("[%s] Found out of order transactions", safe_address)
+            self.fix_out_of_order(
+                safe_address,
+                InternalTxDecoded.objects.pending_for_safe(safe_address)[0].internal_tx,
+            )
+            self.tx_processor.clear_cache(safe_address)
+
+        # Use chunks for memory issues
+        number_processed = 0
```

Comment:
```suggestion
        total_processed_txs = 0
```
For me number processed could indicate an ID for example, not the total amount of processed transactions.